packages/runtime/src/extensibility/facades/transport/PublicRelationshipTemplateReferencesFacade.ts

```diff
@@ -0,0 +1,12 @@
+import { Result } from "@js-soft/ts-utils";
+import { Inject } from "@nmshd/typescript-ioc";
+import { PublicRelationshipTemplateReferenceDTO } from "../../../types/transport/PublicRelationshipTemplateReferenceDTO";
+import { GetPublicRelationshipTemplateReferencesUseCase } from "../../../useCases/transport/publicRelationshipTemplateReferences/GetPublicRelationshipTemplateReferences";
```

Comment:
missing barrels

Reply:
This would result in circular deps that why I use the direct imports here

Reply:
I don't know why it should result in circular dependencies here but nowhere else.

Reply:
Ok, we can use `type` and `use cases` but that actually is not lower the import count

Reply:
that's not the point at all. We want to make it easy to add new usecases, you just made it easy for yourself.

Reply:
Ah, ok i mostly only use auto import, so I was not aware